src/components/confirm-dialog/confirm-dialog.js

```diff
@@ -0,0 +1,44 @@
+import template from "./confirm-dialog.html";
+import { MDCDialog } from "@material/dialog";
+
+export class ConfirmDialogComponent {
+  constructor(mountPoint, props) {
+    this.mountPoint = mountPoint;
+    this.props = props;
+  }
+
+  showDialog() {
+    this.dialog.show();
+  }
+
+  handleOkClick() {
+    this.props.onOkClicked();
+  }
+
+  handleCancelClick() {
```

Comment:
probably we do not need to handle `cancel` case -- so please remove these handlers